oci-artificial-intelligence/ai-language/custom-model/files/customTXTCPythonSDK.py

```diff
@@ -0,0 +1,99 @@
+import time
+import oci
+print(f'OCI Client SDK version: {oci.__version__}')
+config = oci.config.from_file()
+
+compartment_id = "ocid1.tenancy.oc1..aaaaaaaaih4krf4od5g2ym7pffbp6feof3rx64522aoxxvv3iuw3tam6fvea"
```

Comment:
remove ocid
declare variables at begning for object storage namespace,e tc

Reply:
Done